data/meta/strategies/1/LiquityFeeClaimReinvest.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "strategy",
+  "name": "Liquity Fee Claim and Reinvest",
+  "description": "Accepts {{token}} and stakes it at [Liquity](https://liquity.app/) to earn LUSD and ETH. Earned rewards are harvested, swapped for more {{token}} which is deposited back into the strategy. Native rewards are increased via yLQTY Booster, which donates yield from staking Yearn's protocol-owned LQTY.",
```

Comment:
2nd sentence of description, first part.  

`Earned, rewards are harvested, swapped `

Feels a bit strange my intuition says either "then" or "and" should be added after "harvested" let me know what you think though

Reply:
While I do agree with you its a standard phrasing we have used on other strategies and I think for now we will stick with it.